src/components/ui/PopOver/index.tsx

```diff
@@ -40,6 +40,8 @@ export interface PopOverProps
   renderButton?: (isActive: boolean) => {};
   /** Alternative to renderButton, defines the function to render the full popover button element (as opposed to just its contents). This is used if you want full control over the button rendering. The button must forwardRef */
   renderButtonWrapper?: (isActive: boolean, props: any) => {};
+  /** The callback fired when the render button is clicked. */
```

Comment:
Is this on the render button click or the PopOver element click?

Reply:
It is the "^" button click.